src/features/content-explorer/content-explorer-modal-container/ContentExplorerModalContainer.js

```diff
@@ -13,6 +13,8 @@ class ContentExplorerModalContainer extends Component {
          * Each column has to be a Column element
          */
         additionalColumns: PropTypes.arrayOf(PropTypes.element),
+        /**  Should user be able to choose empty selection in MULTI_SELECT mode, defaults to false */
+        allowChoosingEmptySelection: PropTypes.bool,
```

Comment:
Can we change the name of this prop using a similar convention to other props (e.g. `isSelectAllAllowed`) in this component? I think `isNoSelectionAllowed` might be more appropriate

Reply:
I don't mind `isSelectNoneAllowed` but I think the difference between this and `isSelectAllAllowed` is that there's also **choose** action involved - not only selecting.